src/testdir/test_vim9_script.vim

```diff
@@ -2510,6 +2510,13 @@ def Test_for_loop()
         reslist->add('x')
       endfor
       assert_equal(['x', 'x', 'x'], reslist)
+
+      # Test for ignoring loop var assignment
```

Comment:
is that comment correct?

Reply:
Yes.  The `_` is used to ignore the for loop variable assignment.